nemo_text_processing/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
sorry please don't change files you didn't newly write, like this init file

Reply:
What about for pytests? Should I update them or leave them at 2021?